Makefile

```diff
@@ -35,15 +38,23 @@ database :
 
 ## superuser    : make a super-user in the database
 superuser :
-	@${PYTHON} manage.py create_superuser
+	@${MANAGE} create_superuser
 
 ## airports     : display YAML for airports
 airports :
-	@${PYTHON} manage.py export_airports
+	@${MANAGE} export_airports
 
 ## badges       : display YAML for badges
 badges :
-	@${PYTHON} manage.py export_badges
+	@${MANAGE} export_badges
+
+## check-urls   : check workshop URLs
+check-urls :
+	@${MANAGE} check_workshop_urls ${SITE_PATH}
+
+## check-badges : check that all badges have been awarded
+check-badges :
+	@${MANAGE} check_badges ${SITE_PATH}
```

Comment:
What's `SITE_PATH`? Do I have to run `SITE_PATH=../site make check-badges`? Can make pass down it's args?